pbm/backup/backup.go

```diff
@@ -603,13 +612,16 @@ func (d *mdump) WriteTo(w io.Writer) (int64, error) {
 		InputOptions:    &mongodump.InputOptions{},
 		SessionProvider: &db.SessionProvider{},
 		OutputWriter:    w,
-		ProgressManager: progress.NewBarWriter(os.Stdout, time.Second*3, 24, false),
+		ProgressManager: pm,
 	}
 	err := mdump.Init()
 	if err != nil {
 		return 0, errors.Wrap(err, "init")
 	}
-	return 0, errors.Wrap(mdump.Dump(), "make dump")
+	pm.Start()
+	err = mdump.Dump()
+	pm.Stop()
```

Comment:
It's not improve anything in this code but `defer pm.Stop()` will look more Go-like and save us from bugs if we will have more code between `pm.Start()` and `pm.Stop()` in future

Reply:
At first, I did with the defer and then didn't quite like how looks. But, yes, maybe you're right.